Makefile

```diff
@@ -90,6 +90,10 @@ else
 	$(error CLOUD_PROVIDER is not set)
 endif
 
+.PHONY: provisioner-cli
```

Comment:
I think having a separate Makefile inside `test/tools` will be more apt and give flexibility to add targets fpr new tools as well as any tests for the tools itself.

Reply:
> I think having a separate Makefile inside `test/tools` will be more apt and give flexibility to add targets fpr new tools as well as any tests for the tools itself.

Thanks @bpradipt sounds also better to move readme to tools root directory.

Reply:
Right, both README and a separate Makefile in `tools` root directory makes sense to me.
Thanks @huoqifeng